sql/src/main/java/io/crate/analyze/symbol/Aggregations.java

```diff
@@ -35,6 +39,47 @@ public static boolean containsAggregation(Symbol s) {
         return AGGREGATION_SEARCHER.process(s, null);
     }
 
+    /**
+     * @return true if the symbol is an aggregation or function which contains an aggregation or a scalar.
+     */
+    public static boolean containsAggregationOrscalar(Symbol s) {
+        return AGGREGATION_OR_SCALAR_SEARCHER.process(s, null);
+    }
+
+    /**
+     * @return false if the symbol is not found in the group by, or the if one the symbol's non-literal arguments are not found in the group by.
```

Comment:
`or the if one the symbol's`?

Maybe rename the method to something like `containedInGroupBy` (in case I understand correctly what it does)